packages/templates/cra-template-web-viewer/template/src/index.tsx

```diff
@@ -12,6 +12,8 @@ import App from "./App";
 import { Auth } from "./Auth";
 import * as serviceWorker from "./serviceWorker";
 
+globalThis.IMJS_URL_PREFIX = process.env.IMJS_URL_PREFIX || "";
```

Comment:
remove, dont want to expose this to users, they should always point to prod

Reply:
removed